.stylelintrc.json

```
@@ -0,0 +1,17 @@
+{
+  "extends": "stylelint-config-airbnb",
+
+  "rules": {
+    "indentation": [
+      2,
```

```suggestion
      4,
```

Ну, я опять про свое🙃

---

ок)